db/migrate/20220630092144_add_userid_to_magazine.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# app/model/magazine.rb
+class AddUseridToMagazine < ActiveRecord::Migration[6.0]
+  def change
+    add_column :magazines, :user_id, :integer
```

Comment:
@ShivaniKandukuri whenever you're going to add relationships like this, use add_references, so it will generate **big_int** instead of the the **integer**. See the difference between integer and bigint.